SwrveSDK/SDK/Track/Swrve.m

```diff
@@ -1417,6 +1449,8 @@ -(void) shutdown
     }
 
     [self setEventBuffer:nil];
+    [[self eventStream] removeFromRunLoop:[NSRunLoop currentRunLoop]
```

Comment:
I think that should only be the case if we have called scheduleInRunLoop, which we don't seem to do